index.js

```diff
@@ -6,12 +6,39 @@ module.exports = {
   extends: ['airbnb'],
   plugins: ['chai-friendly'],
   rules: {
-    'no-unused-expressions': 0,
-    'chai-friendly/no-unused-expressions': 2,
+    'no-unused-expressions': 'off',
+    'chai-friendly/no-unused-expressions': 'error',
     'linebreak-style': ['error', 'unix'],
     'no-param-reassign': ['error', { props: false }],
     'func-style': ['error', 'declaration', { allowArrowFunctions: true }],
     'no-use-before-define': ['error', { functions: false }],
-    'consistent-return': 0,
+    'no-unused-vars': ['error', {'vars': 'all', 'args': 'none'}],
```

Comment:
I'm mostly fine with this. I see a small benefit in checking args (makes sure you're using that `wfCb` you declared), but I also understand the reasoning behind changing this (method signatures I assume?).

I do wonder if the pros of this rule outweigh the cons. Unit testing should help identify where we haven't used something we should. Will defer to your judgement.

Reply:
We don't necessarly use `wfCb` as far as i'm concerned : the last step can (maybe shoudln't, but that's another debate) call `done` directly
Keeping, mostly like you said, for framework prototype; or overwriting on /server

Reply:
I was just thinking of scenarios of callbacks within callbacks within callbacks and not realising we've called the wrong callback.

That said, callbacks within callbacks within callbacks is just bad design that we should avoid, and Promises async/await should help a good bit with that. That and just smaller simpler functions.

So, overall 👍